front/src/applications/stdcm/views/StdcmRequestModal.tsx

```diff
@@ -20,14 +19,15 @@ import createTrain from 'modules/simulationResult/components/SpaceTimeChart/crea
 import formatStdcmConf from 'applications/stdcm/formatStcmConf';
 // Static Data and Assets
 import { setFailure } from 'reducers/main';
-import { OsrdStdcmConfState } from 'applications/operationalStudies/consts';
 import STDCM_REQUEST_STATUS from 'applications/stdcm/consts';
-import { updateItinerary } from 'reducers/osrdconf';
 import { useTranslation } from 'react-i18next';
 import { Spinner } from 'common/Loader';
 import { osrdEditoastApi, SimulationReport } from 'common/api/osrdEditoastApi';
 import { StdcmRequestStatus } from 'applications/stdcm/types';
 import { extractMessageFromError, extractStatusFromError } from 'utils/error';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { OsrdStdcmConfStateType, stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you also update the other type imports of the file please
```suggestion
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { OsrdStdcmConfStateType, stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done

Reply:
Could you also import type `SimulationReport`

Reply:
Done